### DISCUSSION TITLE
More examples

### DISCUSSION THREAD

**Discussion by NicolasSacC** [filamentphp/filament#4744] [Ideas]
There is a lot of examples throughout the code except for the most difficult parts such as https://filamentphp.com/docs/2.x/forms/fields#building-custom-fields or https://filamentphp.com/docs/2.x/admin/resources/custom-pages. 

I think the documentation would really benefit from more examples, especially for the things that we don't have to do often but will happen in a web app. (like custom fields, a small example on how to set the data or a link to some other documentation would be great).

**Comment by Z3d0X** [comment]
I think you may find Ryan Chandler's series on building a [Color Palette Field](https://github.com/ryangjchandler/filament-color-palette) helpful
- [Part 0](https://ryangjchandler.co.uk/posts/building-a-custom-color-palette-field-in-filament-part-0)
- [Part 1](https://ryangjchandler.co.uk/posts/building-a-custom-color-palette-field-in-filament-part-1)
- [Part 2](https://ryangjchandler.co.uk/posts/building-a-custom-color-palette-field-in-filament-part-2)
- [Part 3](https://ryangjchandler.co.uk/posts/building-a-custom-color-palette-field-in-filament-part-3)
